src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/ElectricDustWasher.java

```diff
@@ -83,13 +84,19 @@ else if (ChargableBlock.isChargable(b)) {
 			for (int slot: getInputSlots()) {
 				if (SlimefunManager.isItemSimiliar(BlockStorage.getInventory(b).getItemInSlot(slot), SlimefunItems.SIFTED_ORE, true)) {
 					boolean empty_slot = false;
```

Comment:
the empty_slot boolean should be moved inside the (!legacy_dust_washer) statement as it is not needed otherwise.